Forge.TreeWalker/src/TreeNodeContext.cs

```diff
@@ -0,0 +1,100 @@
+//-----------------------------------------------------------------------
+// <copyright file="TreeNodeContext.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The TreeNodeContext class.
+// </summary>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Forge.TreeWalker
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
```

Comment:
Can we remove these? I don't see any Tasks in this file.

Reply:
Good catch! Pretty sharp eye in code view.

---
In reply to: [504221641](https://github.com/microsoft/Forge/pull/32#discussion_r504221641) [](ancestors = 504221641)